nixos/modules/hardware/uinput.nix

```
@@ -0,0 +1,19 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.hardware.uinput;
+in {
+  options.hardware.uinput = {
+    enable = lib.mkEnableOption "Whether to enable uinput support";
```

`mkEnableOption` already prefixes the argument:

https://github.com/NixOS/nixpkgs/blob/2ce7e4ffb7893a0dd74c5f346cff3560db6dc930/lib/options.nix#L63-L68

```suggestion
    enable = lib.mkEnableOption "uinput support";
```